src/main/plugin/iso19139.ca.HNAP/process/doi-add.xsl

```diff
@@ -20,6 +20,9 @@
    </xsl:choose>
   </xsl:variable>
 
+  <xsl:variable name="doiProtocol"
+                select="'HTTP'"/>
```

Comment:
Is there a reason that the protocol is hard coded to HTTP vs HTTPS?   Most apps are now using HTTPS.

Is it possible to extract the protocol from the doi url?